dataset_examples/laion-art-ignore-directives.md

```diff
@@ -5,7 +5,9 @@ See [full description](https://github.com/LAION-AI/laion-datasets/blob/main/laio
 
 It is available at https://huggingface.co/datasets/laion/laion-art
```

Comment:
please add a few lines to explain how keeping disallowed_header_directives to the default of ["noai", "noimageai", "noindex", "noimageindex"] will help 1% of humans keep their privilege and exclude 99% of humans the ability to create art.

Reply:
This seems a little snarky. 

If you want to draw anything right now, you have created art. This won't change that. 

I don't understand how this changes how many people can make art? This won't gatekeep the people who want to ignore artists anyway: they can just disable the feature.

Reply:
Trying to slow down the creation of image generation image may slow down access of these models to people.

Most people do not have a talent for art. Such generative models allow them to create art.

So yes, removing access to artistic picture from model training sets will help artists keep their privilege and prevent most people from creating art.

Reply:
Okay, but most people also do not have the talent for playing football at a professional level. 

Are we preventing them from playing football? No, you can play football. You just aren't good enough to play on the National Team. 
You can play football or create art without being a professional, nobody is stopping you. It's one of the cheapest hobbies out there. 

Anybody can create art. It's not a new invention of 2022. Most artists who work professionally have been drawing for 10+ years to become skilled, it's not a talent they are born with. 

If you're saying that these image generation models are only viable and can only create art when using images by artists, are you not inherently saying that they need to art to work?

Again, this PR is only about the DEFAULT. You can (and probably will) ignore it.